core/src/main/kotlin/org/evomaster/core/output/sorting/SortingStrategy.kt

```diff
@@ -0,0 +1,11 @@
+package org.evomaster.core.output.sorting
+
+enum class SortingStrategy {
+
+    COVERED_TARGETS,
+    TARGET_INCREMENTAL
+    ;
+
+    fun isCoveredTargets() = this.name.equals("covered_targets", true)
+    fun isTargetIncremental() = this.name.equals("target_incremental", true)
```

Comment:
is this needed? we have something similar in `OutputFormat`, but, there, more than 1 enum value could make a predicate true. if it is an exact match, why not checking the enum value directly?

Reply:
Yep, you're right